api_tests/preprints/views/test_preprint_list.py

```diff
@@ -9,6 +11,7 @@
 from osf.models import PreprintService, Node
 from website.project import signals as project_signals
 import mock
+from tests.json_api_test_app import JSONAPITestApp
```

Comment:
I don't often do this perfectly, but since we are reworking a lot of files now is also a good time to reorder imports. Take a look at http://cosdev.readthedocs.io/en/latest/style_guides/python.html#imports and see if any of the files you update need to be reordered.